schema.graphql

```diff
@@ -2329,3 +2358,120 @@ type CustomClaimType @entity {
   createdBlock: Block!
   updatedBlock: Block!
 }
+
+enum AffirmingPartyEnum {
+  Sender
+  Receiver
+  Mediator
+}
+
+enum AffirmStatusEnum {
+  Affirmed
+  Rejected
+}
+
+enum TransactionStatusEnum {
+  Created
+  Executed
+  Rejected
+}
+
```

Comment:
"""
Represents a Confidential Transaction sender proof of funds
"""

And comments for the other types. Not sure how much they add, but it will look more consistent